src/main/java/org/opentripplanner/updater/configure/UpdaterConfigurator.java

```diff
@@ -166,9 +165,6 @@ private List<GraphUpdater> createUpdatersFromConfig() {
     for (var configItem : updatersParameters.getSiriSXUpdaterParameters()) {
       updaters.add(new SiriSXUpdater(configItem, transitModel));
     }
-    for (var configItem : updatersParameters.getSiriVMUpdaterParameters()) {
```

Comment:
Siri VM is realttime positions isn't it? What is the reason for its removal?

Reply:
We don't have support for reading in positions, but rather updating the timetable based on the delay value, similar to how you could update the last delay from a GTFS-RT vehicle position. We don't have need for this any more, since all of the producers now have a SIRI-ET feed. The removal was done in two commits, first the creation of timetable updates was removed, and the updater infrastructure in a second commit, so that only that commit can be reverted, if reading of vehicle positions is added some time in the future.

Reply:
It's likely that one of my clients wants to use Siri-VM in the future but not for creating timetable, just vehicle positions. In such a case I might undelete this code.

Reply:
Commit https://github.com/opentripplanner/OpenTripPlanner/commit/f722146b977930ffcae9226a04dce80fbad5e445 contains all the infrastructure, which should bought back for implementing that.